Gemfile.lock

```diff
@@ -1,3 +1,10 @@
+GIT
+  remote: https://github.com/adiwg/mdJson-schemas.git
+  revision: 0ced67aab4ab2054682289acd86e781e1900b2a0
+  branch: bugfix/data-quality-evaluation-method
+  specs:
+    adiwg-mdjson_schemas (2.8.1)
+
```

Comment:
If we need to update the Gemfile.lock I'll look into it. I know part of this is because of the Gemfile change, so this needs to be reverted back also.